gen3_tracker/git/cli.py

```
@@ -1159,16 +1169,21 @@ def rm(config: Config, object_id: str):
                 for dvc_object in dvc_objects
                 if dvc_object.object_id == object_id
             ]
-            assert dvc_objects, f"{object_id} not found in MANIFEST."
-            dvc_object = dvc_objects[0]
-            path = pathlib.Path("MANIFEST") / (dvc_object.out.path + ".dvc")
-            assert path.exists(), f"{path} not found"
-            path.unlink()
-        click.secho(
-            f"Deleted {path} from MANIFEST. Please adjust META resources",
-            fg=INFO_COLOR,
-            file=sys.stderr,
-        )
+
+            if not dvc_objects:
+                assert path.exists(), f"{object_id} {path} not found in MANIFEST path or in commited files."
```

typo: committed with two t's

---

Also James was telling that [asserts get ignored](http://docs.astral.sh/ruff/rules/assert/) in some production environments, should we raise an exception instead?

---

addressed typo.
left asserts - open a new issue so we can make change en-masse